Makefile

```
@@ -22,3 +22,10 @@ helm-reinstall: helm-uninstall helm-install
 
 helm-deps-up: validate-k-env
 	helm dependency update ./charts/tensorleap
+
+.PHONY: deploy-local
+deploy-local:
```

You must do this in order to make sure you're not accidentally breaking the cloud environment you have in the current `kubectl` context
```suggestion
deploy-local: validate-k-env
```

also:
1. I'm not sure if `deploy-local` is the correct name, it's not really like the other deployment
2. I wish @roee-shapira-tensorleap to talk a look as well, maybe is collides with something
3. Worth writing about it somewhere in slack after merging
4. Next time it's worth taking the time to put stuff like this in a separate PR, it's not relevant to the subject _Update es query bucket limit_, and now we can't merge the actual change that is great and could have been just approved quickly